content/en/docs/tutorial/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Tutorials
```

Comment:
After our latest doc WG discussion about examples, I'm thinking that rather than having a top level Tutorials folder, this would instead be better in subfolder of the Examples section. The example title could be something like "Istio Tutorial", or maybe something like "Istio 101", or "Istio End-to-end". Under that would be each module in order.

Reply:
Ah, I just realized that if we move this tutorial to Examples, we can just get rid of this top level Tutorials folder. I think we should do that for now. After we have a better reorg of everything else in Examples, we can add this back if appropriate, but I'm thinking that this tutorial will fit well in the Examples section once we clean it up.

Reply:
Fixed